src/Propel/Generator/Command/MigrationDiffCommand.php

```diff
@@ -100,7 +100,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $manager->setWorkingDirectory($generatorConfig->getSection('paths')['migrationDir']);
 
         if ($manager->hasPendingMigrations()) {
-            throw new RuntimeException('Uncommitted migrations have been found ; you should either execute or delete them before rerunning the \'diff\' task');
+            throw new RuntimeException(sprintf(
+                'Uncommitted migrations %s have been found ; you should either execute or delete them before rerunning the \'diff\' task',
+                implode(', ', $manager->getValidMigrationTimestamps())
+            ));
```

Comment:
I like this! Though I would prefer it if the names of open migrations were at the end of the error message, preferably each on their own line, so I don't have to search for them somewhere within the error message.

The name `->getValidMigrationTimestamps()` is obviously crap, and I would change it, even though it has nothing to do with the purpose of the PR. But I don't know if people like me more or less for that...

Reply:
Hey @Deltachaos, didn't realize you updated this. Nice!
You can fix the formatting errors automatically by running `composer run-script cs-fix src/Propel/Generator/Command/MigrationDiffCommand.php`.
Small nitpick though: Using `sprintf` for concatenation is unnecessary. Why not simply do regular concatenation?